docs/networks/node-ops/node-operation/node-bootstrap.md

```diff
@@ -235,16 +237,20 @@ This command will detect and provide information about common misconfigurations,
 
 ### Push transit keys (consensus node only)
 
-If you are running a consensus node, run the following command to push the transit keys. Use the token `mainnet-x-organization` e.g. `mainnet-17-companyname`.
+If you are running a consensus node, run the following command to generate the transit keys.
 
 ```shell transit
-$ ./boot-tools/transit push-transit-key -b ./bootstrap -t mainnet-x-organization
-<nil> INF generating transit keys
+$ ./boot-tools/transit prepare -b ./bootstrap -r consensus
+<nil> INF running prepare
 <nil> INF generating key pair
-<nil> INF attempting to push transit public key to the transit servers
-<nil> INF successfully pushed transit public key to the transit servers
+<nil> INF completed preparation role=consensus
 ```
 
+This will generate the public and private transit keys under the bootstrap folder.
+The transit keys are used to transfer the DKG keys after a network upgrade.
+
+Please share with the FlowFoundation the **public** transit key via email or discord.
```

Comment:
```suggestion
Please share the **public** transit key with the Flow Foundation via email or discord.
```
Adjusted for wording. Can we provide links for email and discord here?

Reply:
added link to discord and the email address.